### DISCUSSION TITLE
aide et incompréension

### DISCUSSION THREAD

**Discussion by arthurcadot** [photoprism/photoprism#4257] [Q&A]
J'ai besoin d'aide quelqu'un est disponible ?

**Comment by arthurcadot** [comment]
J'ai instaler photoprism par yunohost, mais je n'arrive pas a trouver le dossier frontend dans mon serveur ?

**Reply by graciousgrey** [reply]
Hello :wave: ,

The yunohost integration is not maintained by us, maybe another yunohost user can help here.

**Comment by arthurcadot** [comment]
J'ai avancer le dossier n'y est pas mais j'ai trouver world.json que j'ai remplacer avec les meme permision et proprietaire mais je ne vois pas de changement sur mon application en ligne ?
